lua/apg/cl_menu.lua

```diff
@@ -93,13 +93,13 @@ local function APGBuildGhostPanel()
 	end
 	utils.switch( panel, 0, 180, 170, 20, "Always frozen", "alwaysFrozen" )
 	utils.switch( panel, 0, 215, 170, 20, "Apply to doors", "fadingDoorGhosting" )
-	utils.switch( panel, 0, 250, 170, 20, "Ignore Vehicles", "dontGhostVehicles" )
+	utils.switch( panel, 0, 250, 170, 20, "Ignore Vehicles", "vehAntiGhost" )

```

Comment:
In the same vein, why not just `vehicleAntiGhost` ?

Reply:
Mainly because it's already in the config as "vehDamage", "vehNoCollide", "vehIncludeWAC", "vehAntiGhost" might change it later but if you read the config it's self-explanatory.

Reply:
No I think consistency is important - good choice!